src/Round.js

```diff
@@ -1,42 +1,41 @@
 const Turn = require('../src/Turn');
-const game = require('../src/Game');
 const data = require('./data');
 const prototypeQuestions = data.prototypeData;
 const util = require('./util');
 const Deck = require('./Deck');
 
 class Round {
-  constructor(deck, turn) {
+  constructor(deck) {
     this.deck = deck.deck;
     this.turns = 0;
     this.incorrectGuesses = [];
     }
-    returnCurrentCard = (card) => {
-      return this.deck[0];
+    returnCurrentCard = () => {
+      return this.deck[0]
     }
-    takeTurn = (guess) => {     
+    takeTurn = (guess) => {   
       let currentCard = this.returnCurrentCard();
-      let newTurn = new Turn(guess, currentCard);
-      !newTurn.evaluateGuess() ? this.incorrectGuesses.unshift(currentCard.id) : null;
+      let turn = new Turn(guess, currentCard);
+      let result = turn.evaluateGuess();
+      !result ? this.incorrectGuesses.unshift(currentCard.id) : null;
       this.deck.shift();
       this.turns++;
-      return newTurn.giveFeedback();
+      return turn.giveFeedback()
     }
     calculatePercentCorrect = () => {
-      return  Math.round((this.turns - this.incorrectGuesses.length) / this.turns* 100 ) ;
+      return  Math.round((this.turns - this.incorrectGuesses.length) / this.turns * 100 ) ;
     }
     bonusRound = () =>{
-      console.log(`**Bonus Round!!!**, redemption time!`)
-      let newBigDeck = []
       this.incorrectGuesses.forEach(id =>{
-       newBigDeck.push(prototypeQuestions.filter(x => x.id === id))
-      })
-      newBigDeck = new Deck([].concat(...newBigDeck))
-      let newestRound = new Round(newBigDeck)
-      util.main(newestRound)
+        this.deck.push(prototypeQuestions.filter(x => x.id === id));
+     })
+      this.deck = new Deck([].concat(...this.deck));
+      let bonusRoundDeck = new Round(this.deck);
+      util.main(bonusRoundDeck);
```

Comment:
This is the final form for the bonusRound method, it is functional and SRP compliant